packages/cli/test/integration/workflows/workflows.controller.ee.test.ts

```diff
@@ -428,28 +428,38 @@ describe('GET /workflows/:workflowId', () => {
 		expect(responseWorkflow.sharedWithProjects).toHaveLength(0);
 	});
 
-	test('should return workflow with credentials saying owner does not have access when not shared', async () => {
-		const savedCredential = await saveCredential(randomCredentialPayload(), { user: member });
+	test.each([
```

Comment:
This converted the existing test rather than adding to it. I'd expect we'd want to keep a test for the member scenario?

Reply:
I think the next test is covering the scenario for members. 

https://github.com/n8n-io/n8n/blob/7f312ab6540e134dfba1a4bf7a889c7772040449/packages/cli/test/integration/workflows/workflows.controller.ee.test.ts#L464-L499

This test was only checking the owner scenario.